client/src/main/java/org/spine3/base/Events.java

```diff
@@ -82,25 +78,19 @@ public static void sort(List<Event> events) {
         Collections.sort(events, EVENT_TIMESTAMP_COMPARATOR);
     }
 
-    /**
-     * Obtains the timestamp of the event.
-     */
+    /** Obtains the timestamp of the event. */
     public static Timestamp getTimestamp(Event event) {
         final Timestamp result = event.getContext().getTimestamp();
         return result;
     }
 
-    /**
-     * Creates a new {@code Event} instance with the passed event message and context.
-     */
+    /** Creates a new {@code Event} instance with the passed event message and context. */
     @SuppressWarnings("OverloadedMethodsWithSameNumberOfParameters")
     public static Event createEvent(Message event, EventContext context) {
         return createEvent(toAny(event), context);
     }
 
-    /**
-     * Creates a new {@code Event} instance with the passed event message wrapped to {@link Any} and context.
-     */
+    /** Creates a new {@code Event} instance with the passed event message wrapped to {@link Any} and context. */
```

Comment:
I guess it is too long for the summary. Pls leave only `Creates a new {@code Event} instance` on the first line. And do the same where possible.